pandas/tests/indexes/interval/test_interval.py

```diff
@@ -886,6 +886,13 @@ def test_symmetric_difference(self, closed, sort):
         result = index.symmetric_difference(other, sort=sort)
         tm.assert_index_equal(result, expected)
 
+    def test_interval_range_get_indexer_with_different_input_type(self):
+        # not sure about this one
+        index = pd.interval_range(0, 1)
+        # behaviour should be the same as Int64Index and return an
+        # array with values of -1
+        assert np.all(index.get_indexer(['gg']) == np.array([-1]))
```

Comment:
use `tm.assert_numpy_array_equal` and make sure your expected is `'intp'` dtype.